northstar-tests/Cargo.toml

```diff
@@ -25,4 +24,4 @@ uuid = { version = "0.8.2", features = ["v4"] }
 [build-dependencies]
 tempfile = "3.2"
 escargot = "0.5.2"
-npk = { path = "../npk" }
+northstar = { path = "../northstar", features = ["runtime"] }
```

Comment:
Shouldn't this need also feature `npk` because the `test-container` is built from the `build.rs`?

Reply:
I replaced "runtime" with "npk" as runtime was not really needed.